release-notes/2023-04/README.md

```diff
@@ -0,0 +1,239 @@
+# SystemLink Enterprise 2023-04 Release Notes
+
+The 2023-04 release for SystemLink Enterprise has been published to <https://niedge01.jfrog.io>. This update includes new features, bug fixes, and security updates. Work with your account representative to obtain credentials to access these artifacts. If you are not upgrading from the previous release, refer to past release notes to ensure you have addressed all required configuration changes.
+
+## Upgrading from the 2023-03 to the 2023-04 release
+
+- JupyterHub user pod PVCs incorporate a user's email address in the PVC name.
```

Comment:
Don't think this needs to be a list since it's all really about one thing

Reply:
Will do